staging/src/k8s.io/api/certificates/v1beta1/types.go

```diff
@@ -132,11 +133,16 @@ type RequestConditionType string
 const (
 	CertificateApproved RequestConditionType = "Approved"
 	CertificateDenied   RequestConditionType = "Denied"
+	CertificateFailed   RequestConditionType = "Failed"
 )
 
 type CertificateSigningRequestCondition struct {
-	// request approval state, currently Approved or Denied.
+	// type of the condition. Known conditions include "Approved", "Denied", and "Failed".
 	Type RequestConditionType `json:"type" protobuf:"bytes,1,opt,name=type,casttype=RequestConditionType"`
```

Comment:
:eyes: we don't have `// +optional` here - doesn't this mean this field is already required with newer kubectl versions? (but I guess not with API clients)

Reply:
this hasn't changed, so... yes... anything submitting a condition using openapi validation would need to include the field, but they could set it's value to an empty string